### ISSUE TITLE
Can I use this with a config file?

### ISSUE THREAD

**Issue by fgiraldi** [ctrf-io/newman-reporter-ctrf-json#14]
I have a TypeScript file where I run a Postman collection via Newman. I already generate an Allure report and a json report (which doesn't meet my needs), but I need some json so that I can report to another API. I saw the output produced by this package is exactly what I am looking for, but I don't know how to handle its parameters from the code.

I have this code to generate 3 reports (Allure, traditional json and ctrf-json) but I want to configure the output file to be inside the api_reports folder. 
```
import { NewmanRunOptions, run } from 'newman';

const options: NewmanRunOptions = {
    collection,
    reporters: ['allure', 'json', 'ctrf-json'],
    reporter: {
        ctrfJson: {export: 'api_reports/api_report.json'},
    },
    environment,
    delayRequest: 1000,
};
```

I see there is no documentation for config files, only CLI options. Does this support in-code configuration?
If so, I might suggest adding that documentation to the README

BTW: I have already tried:
```
    reporter: {
        json: {export: 'api_reports/api_report.json'},
        ctrfJson: {export: 'api_reports/api_report.json'},
    },

```
```
    reporter: {
        json: {export: 'api_reports/api_report.json'},
        "ctrf-json": {"output-file": 'api_reports/api_report_ctrf.json'}
    },
```
and many other combination with no success

**Comment by Ma11hewThomas** [comment]
Thanks for reporting, published with instructions in v0.0.7

**Comment by fgiraldi** [comment]
Worked like a charm! Thanks!
